src/client.jsx

```diff
@@ -14,11 +16,14 @@ addLocaleData(fr)
 export function renderApp() {
   const appMountNode = document.getElementById("app-mount-node")
   const initialState = window.__INITIAL_STATE__
+  const history = config.piwikConfig
```

Comment:
I changed the config files to export a `null` key explicitly if piwikConfig is not relevant.